CHANGELOG/CHANGELOG-1.1.md

```diff
@@ -0,0 +1,47 @@
+# v1.1.0 - Changelog since v1.0.0
+
+## Changes by Kind
+
+## Feature
+
+- Improved Windows Support
+  - Update driver to use CSI proxy beta for Windows ([#607](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/607), [@jingxu97](https://github.com/jingxu97))
```

Comment:
Is there a specific proxy beta version we depend on? Would be good to specify in the note.

Reply:
all csi proxy beta version can work since there is no api change. The latest one is beta.2 (v0.2.2)

Reply:
@jingxu97

Reply:
Isn't v0.2.2 required because of the new beta apis for volume and disk?c

Reply:
I'll leave it as is then.

Reply:
> Isn't v0.2.2 required because of the new beta apis for volume and disk?

no, the api already exist since 0.2.0. But we make some changes in the implementation.

Reply:
So should we say you can use the driver with csi proxy v0.2.0? or is it still better to use v0.2.2 for the bug fixes you added?

Reply:
good point. Just for this particular PR #607 in change log, if the main purpose is to explaining that PR, then I think update driver to use beta API seems ok, we could also mention the latest version if beta.2 (v0.2.2). For PR #633, we change to use csi_proxy dismount API call. Here I think it is good to mention CSI Proxy v0.2.2 release include data cache flush.

Reply:
Spoke with @jingxu97 to clarify what we want to communicate. Updated doc.